make/development-environments.mk

```diff
@@ -8,12 +8,14 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 .PHONY: install-metallb
-install-metallb: SUBNET_OFFSET=1
+install-metallb: SUBNET_OFFSET=0
```

Comment:
Why is the offset being changed from 0 to 1?
I recall there being some problem with overlapping IPs between clusters and loadbalancers.
Is that still valid? https://github.com/Kuadrant/kuadrant-operator/pull/676

Reply:
Yeah, you're right, i will revert this.